### ISSUE TITLE
Include dayOrNight in scene_list?

### ISSUE THREAD

**Issue by jflasher** [landsat-pds/landsat_ingestor#17]
When we're bringing in nighttime data, the results look a bit odd (e.g., http://landsat-pds.s3.amazonaws.com/L8/140/209/LC81402092016228LGN00/index.html). Is there a way we can make it obvious in the `scene_list` which scenes are NIGHT scenes? Looking for cloudCover of -1 may be a fine proxy, but could be nice to make it more obvious what's going on. This data is already in individual metadata files, just not in top level `scene_list`.

**Comment by warmerdam** [comment]
I agree that this would be desirable as an item in the scene list file.  It is possible to get this from the MTL file of course, but fetching that adds non-trivial extra expense.  However, populating a new flag in the scene_list file is most easily done while recataloging all the scenes, so I'm hesitant to take it on just now.

**Comment by jflasher** [comment]
Sounds good, maybe we can just keep it out there as a nice to have if we end up needing to recatalog all scenes anyways at some point. Will use cloudCover = -1 as proxy for now.
